src/org/openlcb/cdi/swing/CdiPanel.java

```diff
@@ -129,7 +144,10 @@ public void release() {
             task.run();
         }
         cleanupTasks.clear();
-        tabColorTimer.cancel(); 
+        tabColorTimer.cancel();
+        JFrame f = new JFrame();
+        f = (JFrame)SwingUtilities.getAncestorOfClass(f.getClass(), this);
+        f.dispose();
```

Comment:
This call closes the window? If so, we should update the documentation comment for this function.

Reply:
Moved to targetWindowClosingEvent().